pkg/lib/cockpit-components-password.jsx

```diff
@@ -123,7 +123,6 @@ export const PasswordFormFields = ({
                                </button>
                            </Popover>
                        }
-                       validated={error_password ? "warning" : "default"}
```

Comment:
I wonder what the intention was here.  This kinda looks like a `variant` field, except this component doesn't have one.

In any case, this doesn't make anything worse.

Reply:
The `validated` property normally goes into the input element *inside* of the FormGroup, like e.g. [here in kdump](https://github.com/cockpit-project/cockpit/blob/294fd5305c9eefc41cff4d59a06f78a0040c6843/pkg/kdump/kdump-view.jsx#L299). Firewall and lots of other places use that, too.

In this case it was meant to go into the `TextInput className="check-passwords" ` below, but that already has it. So removing it here is correct.